lib/nostrum/shard/payload.ex

```diff
@@ -41,28 +41,14 @@ defmodule Nostrum.Shard.Payload do
   end
 
   @doc false
-  # No stream
-  def status_update_payload(idle_since, game, status, afk, nil) do
+  def status_update_payload(idle_since, {game, stream}, status, afk, type) do
```

Comment:
I believe the intent was to pattern match on the `stream` variable here to check for nil values. This is assuming that we can't send a url param with a nil value. If we can, then this is fine.